src/Model/SnapUser.hs

```diff
@@ -0,0 +1,28 @@
+module Model.SnapUser where
+  
+import Data.Time.Calendar
+import Snap.Snaplet.PostgresqlSimple
+
+data SnapUser = SnapUser
+  { id :: Int
```

Comment:
@kinorius123 The `Prelude` exports a function called `id`. I think this field will `shadow` the function. There will be some conflicts at some point. Could you please rename it to `id_` or `key`?